uutils-coreutils/PKGBUILD

```
@@ -39,24 +40,32 @@ nix.path = "../nix"
 END
 
   cargo update -p nix
-  cargo update -p signal-hook-mio --precise 0.2.5
-  cargo update -p mio@1.0.4 --precise 1.1.0
   cargo update -p libc --precise 0.2.178
   cargo fetch --locked --target "${RUST_CHOST}"
 }
 
 # spliting build() sometimes cause building twice at make build
 package() {
-  cd "${_realname}-${pkgver}"
+  cd "$pkgname"
+  bsdtar -a -cf docs/tldr.zip --files-from /dev/null # Mask many warnings
+  export CARGO_BUILD_BARGET="$pkgname/cargo_build_target" && cargo clean # base image issue?
 
   export RUSTONIG_DYNAMIC_LIBONIG=1
 
   make install \
     DESTDIR="${pkgdir}" \
     PREFIX="/usr" \
     PROFILE=release-fast \
-    SKIP_UTILS="stdbuf" \
-    PROG_PREFIX="uu-" \
+    SKIP_UTILS="kill more uptime" \
+    LIBSTDBUF_DIR="/usr/lib/coreutils" \
```

Avoid conflict with GNU coreutils, and also match Arch.
```suggestion
    LIBSTDBUF_DIR="/usr/lib/uutils-coreutils" \
```